app/views/pieces/show.html.haml

```diff
@@ -1,3 +1,5 @@
+.embed-container
+  %iframe{width: "560", height: "315", src: "http://www.youtube.com/embed/#{@youtube_url}"}
```

Comment:
Line is too long. [91/80]<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace